codegen/gateway.go

```diff
@@ -526,6 +584,36 @@ func NewGatewaySpec(
 		return nil, errors.Wrap(err, "cannot create template")
 	}
 
+	// TODO(sindelar): Filter test files.
+	middlewareFiles, err := filepath.Glob(filepath.Join(
```

Comment:
We may not have any "open source" middlewares, we can remove this if that is the case.